applications/adv_networking_bench/cpp/testing/io_utils.py

```
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)